slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/PlainSQLTest.scala

```diff
@@ -124,4 +124,17 @@ class PlainSQLTest extends AsyncTest[JdbcTestDB] {
       userForIdAndName(2, "foo").map(_.headOption shouldBe None) //TODO Support `head` and `headOption` in Plain SQL Actions
     )
   }
+
+  def testComposableQueries = ifCap(tcap.plainSql) {
+    val create: DBIO[Int] = sqlu"create table USERS(ID int not null primary key, NAME varchar(255))"
+
+    def findUser(name: Option[String], id: Option[Long]) = {
+      val nameQ = name.map{ n => sql"NAME = $n" }
+      val idQ = id.map{ i => sql"ID = $i"}
+      sql"SELECT * FROM USERS " ++ nameQ ++ idQ
```

Comment:
Does that mean that the final SQL is the following? “SELECT * FROM USERS *WHERE* NAME = 'foo' *AND* ID = 42”? What is the logic that introduces the SQL keywords `WHERE` and `AND`?